lib/sassc/native/native_functions_api.rb

```diff
@@ -98,11 +98,6 @@ def self.string_get_type(native_value)
       attach_function "sass_color_set_#{color_channel}".to_sym, [:sass_value_ptr, :double], :void
     end
 
-    # ADDAPI size_t ADDCALL sass_list_get_length(const union Sass_Value* v)
-    # ADDAPI union Sass_Value* ADDCALL sass_list_get_value (const union Sass_Value* v, size_t i);
-    attach_function :sass_list_get_length, [:sass_value_ptr], :size_t
-    attach_function :sass_list_get_value, [:sass_value_ptr, :size_t], :sass_value_ptr
```

Comment:
There are the same definitions here.

https://github.com/sass/sassc-ruby/blob/266c904186545e36e6e3788985c5730306b927d3/lib/sassc/native/native_functions_api.rb#L59-L60

https://github.com/sass/sassc-ruby/blob/266c904186545e36e6e3788985c5730306b927d3/lib/sassc/native/native_functions_api.rb#L53-L54